### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by vladdeSV** [vladdeSV/vscode-klog#17] [general]
First release 🎉 

Features:
- syntax highlighting
- snippets

<hr /><em>This discussion was created from the release <a href='https://github.com/vladdeSV/vscode-klog/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by vladdeSV** [comment]
I made an *oops* and removed the extension and re-released it. It needs to be re-installed for it to update.

@jotaen

**Comment by jotaen** [comment]
Huzza 🥳
